src/templates/ObservabilityPackDetails.js

```diff
@@ -167,48 +147,41 @@ const ObservabilityPackDetails = ({ data, location }) => {
                     )}
                   </>
                 ))}
+                <Intro
+                  css={css`
+                    margin-bottom: 16px;
+                  `}
+                >
+                  {pack.name} observability pack contains{' '}
+                  {pack.dashboards?.length ?? 0} dashboards. These interactive
```

Comment:
We probably want to conditionally render plural or singular language for each type (dashboards, alerts, checks)

Reply:
updated to use a library called `pluralize` if thats okay 🙈 .

Reply:
seems sweet to me!